Extension/src/LanguageServer/configurations.ts

```diff
@@ -1146,6 +1147,9 @@ export class CppProperties {
         const isWindows: boolean = os.platform() === 'win32';
         const config: Configuration = this.configurationJson.configurations[configIndex];
 
+        // Check if config name is unique.
+        errors.name = this.isConfigNameUnique(config.name);
```

Comment:
@sean-mcmanus @Colengms I am checking for all configs that are duplicated here, active ones and inactive ones.

Reply:
I realized something:
![image](https://user-images.githubusercontent.com/16143955/104224262-99f71100-53f9-11eb-8d50-0184cf89fc06.png)
I send an error regarding duplicate names here, but the web view doesn't show the error.

Reply:
A new html element will need to be added to show/hide errors for configuration name. I think the error would be shown when "add configuration" is done or when the current selected configuration (not active one) has a duplicate name.

Reply:
![image](https://user-images.githubusercontent.com/16143955/104240142-74c1cd00-5410-11eb-9ae1-ab01d047a4e1.png)